src/daemon/rpmostreed-transaction-types.c

```diff
@@ -755,6 +832,13 @@ rebase_transaction_execute (RpmostreedTransaction *transaction,
   rpmostreed_transaction_connect_download_progress (transaction, progress);
   rpmostreed_transaction_connect_signature_progress (transaction, repo);
 
+  if (self->revision)
+    {
+      if (!apply_revision_override (transaction, repo, progress, upgrader,
```

Comment:
Something that was a little subtle to me about this is that we're relying on `change_upgrader_refspec()` having been invoked above.  I think that's OK though.